File: C++/First Steps in Coding - Basics of C++/While Loops/Exercise/Moving.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	int width, len, height, space;
	bool freeSpace = true;
	cin >> width; cin >> len; cin >> height;
	space = width * len*height;
	while (true) {
		if (space <= 0) {
			freeSpace = false;
			break;
		}
		string boxes;
		getline(cin >> ws, boxes);
		if (boxes != "Done") {
			int boxesCount = stoi(boxes);
			space -= boxesCount;
		}
		else {
			break;
		}
	}
	if (freeSpace == true) {
		cout << space << " Cubic meters left.";
	}
	else {
		cout << "No more free space! You need " << abs(space) << " Cubic meters more.";
	}
}

File: Python/Basics/While Loop/Exercise/Moving.py
spaceW = int(input())
spaceL = int(input())
spaceH = int(input())

totalSpace = spaceW * spaceL * spaceH

while totalSpace >= 0:
    boxes = input()

    if boxes != "Done":
        boxes = int(boxes)
        totalSpace -= boxes
    else:
        break

if totalSpace < 0:
    print(f"No more free space! You need {abs(totalSpace)} Cubic meters more.")

else:
    print(f"{totalSpace} Cubic meters left.")